Makefile

```diff
@@ -217,7 +220,7 @@ test-job-submit:      ## Submit test training job
 			eval $(minikube docker-env); docker images | grep tensorflow | grep latest > /dev/null || docker pull tensorflow/tensorflow > /dev/null; \
 		fi
 	@node_ip=$$(make --no-print-directory kubernetes-ip); \
-                s3_ip=$$(make --no-print-directory kubernetes-ip); \
+        s3_ip=$$(kubectl get pods | grep restapi- | awk '{print $1}' | xargs -I '{}' kubectl get pod '{}' -o jsonpath='{.status.hostIP}'); \
```

Comment:
Since .status.hostIP in IBM Cloud only shows the internal node IP, can we add a condition and set back the `s3_ip=$$(make --no-print-directory kubernetes-ip)` if the VM_TYPE is none? Thank you in advance.

Reply:
Nevermind, I guess the s3_ip is only used within the cluster. So I believe using the internal IP should be fine.